src/backend/plugins/pacrunner-duktape/pacrunner-duktape.c

```diff
@@ -103,6 +103,17 @@ my_ip_address (duk_context *ctx)
   return duk_error (ctx, DUK_ERR_ERROR, "Unable to find hostname!");
 }
 
+static duk_ret_t
+alert (duk_context *ctx) {
+  if(getenv ("_PX_DEBUG_PACALERT") == NULL)
```

Comment:
Space before ( and i would follow the new rule to set this environment without a leading underscore.

Reply:
I suggested this to write this as !getenv(...) and only use the long checks for numbers.